CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 0.11.3
+
+## New Features
+
+- You can now set a title and a description for PagureIssue.
+
+## Fixes
+
+- GitLab classes can now process more than 20 objects (ogr now understands GitLab's pagination mechanism).
```

Comment:
I would not say that we understand that..;)
https://gitlab.com/gitlab-org/gitlab/-/issues/215641

Reply:
:D ok, I'll reword